spec/component/models/duplicate_publication_group_spec.rb

```diff
@@ -554,6 +554,46 @@
       end
     end
 
+    context 'given a publication with the same DOI as another publication but with different titles and publication date' do
+      let!(:p1) { create :publication,
+                         title: 'A Generic Title',
+                         published_on: Date.new(2000, 1, 1),
+                         doi: 'https://doi.org/10.000/some-doi-22357634' }
+
+      let!(:p2) { create :publication,
+                         title: 'A Different One',
+                         published_on: Date.new(2003, 1, 1),
+                         doi: 'https://doi.org/10.000/some-doi-22357634' }
+
+      it 'groups the publications' do
+        described_class.group_duplicates_of(p1)
+        group = p1.reload.duplicate_group
+
+        expect(group).not_to be_nil
+        expect(p2.reload.duplicate_group).to eq group
+      end
+    end
+
+    context 'given a publication with different DOIs. titles, and publication years' do
+      let!(:p1) { create :publication,
+                         title: 'A Generic Title',
+                         published_on: Date.new(2000, 1, 1),
+                         doi: 'https://doi.org/10.000/some-doi-22357535' }
+
+      let!(:p2) { create :publication,
+                         title: 'A Different One',
+                         published_on: Date.new(2003, 1, 1),
+                         doi: 'https://doi.org/10.000/some-doi-22357534' }
+
+      it 'groups the publications' do
```

Comment:
I think we want 'does not group the publications' here.

Reply:
yep, my bad. I just updated it